spec/capabilities.md

```diff
@@ -0,0 +1,316 @@
+
+### `store/*`
+
+The `store/*` capability can be delegated to a user agent, but cannot be invoked directly. Instead, it allows the audience to derive any capability in the `store/` namespace, provided the resource URI matches the one in the `store/*` capability delegation.
+
+The `store/*` capability (and all capabilities in the `store/` namespace) can be derived from a `*` "super user" capability with a matching resource URI.
+
+### `store/add` 
+
+> Store a CAR file
+
+The `store/add` capability allows an agent to store a CAR file into the memory space identified by the `did:key` URI in the `with` field. The agent must precompute the CAR locally and provide the CAR's CID and size using the `nb.link` and `nb.size` fields, allowing a service to provision a write location for the agent to `PUT` or `POST` the CAR into.
+
+#### Derivations
+
+`store/add` can be derived from a `store/*` or `*` capability with a matching `with` field.
+
+#### Caveats
+
+It is possible for a service to issue a `store/add` delegation with a `link` caveat, which would restrict the user to only storing a specific CID. This is not terribly useful, however, so delegations are unlikely to contain a `link` restriction.
+
+The `size` caveat is much more likely to be included in a delegation, as service providers may want to limit the maximum CAR size that they will accept. Agents should check their delegation's `nb.size` field and ensure that they only send CARs with a size below the limit. If `nb.size` is set in the delegation, the agent must include an `nb.size` field in their invocation, with a value that is `<=` the limit set in the delegation's `nb.size` field.
+
+#### Invocation
+
+Example:
+
+```js
+{
+  can: "store/add",
+  with: "did:key:abc...",
+  nb: {
+    link: "bag...",
+    size: 1234
+  }
+}
+```
+
+Fields marked as "required" below must be present in the invocation, but may be absent in capability delegations. 
+
+| field       | value                             | required? | context                                                             |
+| ----------- | --------------------------------- | --------- | ------------------------------------------------------------------- |
+| `can`       | `store/add`                       | ✅         | The ability to add CAR data to a memory space.                      |
+| `with`      | URI string, e.g. `did:key:123...` | ✅         | The `did:key` URI for the CAR's destination memory space            |
+| `nb.link`   | CAR CID string, e.g. `bag123...`  | ✅         | CID of CAR that the user wants to store                             |
+| `nb.origin` | CAR CID string, e.g. `bagabc...`  | ⛔         | Optional link to related CARs. See below for more details.          |
```

Comment:
Maybe consider the absense of an emjoi here. I'm not sure the stop sign communicates the right meaning.